### DISCUSSION TITLE
After successful connection, display a custom webpage

### DISCUSSION THREAD

**Discussion by nelslar42** [davesteele/comitup#254] [General]
Hi there. I am hoping to simply display a webpage immediately after wifi credentials are successfully entered via comitup. Currently I am prompted properly with comitup, but after it shows "connecting to: SSID name", it does successfully connect to the new network, but then seems to load a white screen. I need a local webpage shown instead of white page. Can anyone help? Please excuse my elementary knowledge here. I have been trying with chatgpt assistance without luck.

**Comment by davesteele** [comment]
The blank page is defined by [this line](https://github.com/davesteele/comitup/blob/7cfc97666c4f131d1be0a33bcd43e3a4c191e4fc/web/templates/connect.html#L6).
